File: src/routes/parametry.js
const routering = require('express').Router();
const { validateToken } = require("../jwt/jeisonWebToken");
const parametry = require('../controllers/parametry')

routering.get('/parametry',validateToken,(req,res)=>{
    parametry.listarParametry(req,res);
})

routering.get('/parametry/data',(req,res)=>{
    parametry.getParametryFront(req,res);
})

module.exports = routering

File: src/routes/parametry.py
from flask import Blueprint, request, jsonify
from controllers.parametry import listarParametry, getParametryFront

routerParametry = Blueprint('routering', __name__)

@routerParametry.route('/parametry', methods=['GET'])
def parametry_route():
    return listarParametry(request, jsonify)

@routerParametry.route('/parametry/data', methods=['GET'])
def parametry_data_route():
    return getParametryFront(request, jsonify)
